varats-core/varats/report/report.py

```diff
@@ -39,6 +39,10 @@ def get_colored_status(self) -> str:
         color."""
         return tp.cast(str, self.status_color[self.name])
 
+    def num_color_characters(self) -> int:
+        """Returns the number of non printable color characters."""
+        return len(re.sub(self.name, '', self.get_colored_status()))
```

Comment:
```suggestion
        return len(self.status_color[''])
```
It's easier to just use the color and a empty string to get the non visible colors